bash_scripts/bash_script/script.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
```

Comment:
add
```
set -e ...

Reply:
figured out how this command works, but why in my script did not understand